docs/resources/Channel.md

```diff
@@ -555,14 +556,15 @@ A thread member is used to indicate whether a user has joined a thread or not.
 
 Embed types are "loosely defined" and, for the most part, are not used by our clients for rendering. Embed attributes power what is rendered. Embed types should be considered deprecated and might be removed in a future API version.
 
-| Type    | Description                                        |
-|---------|----------------------------------------------------|
-| rich    | generic embed rendered from embed attributes       |
-| image   | image embed                                        |
-| video   | video embed                                        |
-| gifv    | animated gif image embed rendered as a video embed |
-| article | article embed                                      |
-| link    | link embed                                         |
+| Type                       | Description                                        |
+|----------------------------|----------------------------------------------------|
+| rich                       | generic embed rendered from embed attributes       |
+| image                      | image embed                                        |
+| video                      | video embed                                        |
+| gifv                       | animated gif image embed rendered as a video embed |
+| article                    | article embed                                      |
+| link                       | link embed                                         |
+| auto_moderation_message    | embed for auto moderation alerts                   |
```

Comment:
"embed containing a message blocked by automod" would be better

Reply:
they won't always contain a message down the line

Reply:
🤔 where else would it operate tho

Reply:
"embed containing blocked content" feels more fitting

Reply:
the current message is fine imo, it's all encompassing no need to restrict the description to containing "blocked content"